.vscode/extensions.json

```diff
@@ -1,17 +1,21 @@
 {
   "recommendations": [
-    "dbaeumer.vscode-eslint",
-    "EditorConfig.EditorConfig",
+    "jasonnutter.vscode-codeowners",
```

Comment:
why it's the only one not in alphabetical order now ? 😃

Reply:
mistake was made